### ISSUE TITLE
Blame splitter between author and content is waaay to the right by default

### ISSUE THREAD

**Issue by mdonatas** [gitextensions/gitextensions#11563]
### Environment

- GitExtensions version: [latest master]

### Issue description

On my machine at some point `Blame` content view "disappeared". In reality it was the splitter by default being on the far right.
Demo:
[ssm-params-custom.webm](https://github.com/gitextensions/gitextensions/assets/483659/c8ab55ab-6207-47b7-985b-29a70c362d03)

I've noticed that the position of this splitter is not saved/restored by the `SplitterManager`... and I have no idea why its position is being set like this.

In my custom build I've fixed it by actually saving the position: https://github.com/gitextensions/gitextensions/commit/1f8ec86bf61452218b29317580ff1d4ed3ee4cde
Not sure I should create a PR for this though as I am pretty sure this is limited to my system somehow... is it working OK for others?

### Steps to reproduce

1. Open GitExtensions
2. Open Diff
3. Blame the file

### Did this work in previous version of GitExtensions?

Yes.. not sure when

**Comment by mstv** [comment]
> Not sure I should create a PR for this though as I am pretty sure this is limited to my system somehow... is it working OK for others?

Yes, please. I have not seen it myself. But there have been reported issues about empty blame. Some of them could be caused by the splitter position, (others by blame taking long time).
